.github/workflows/e2e-test.yml

```diff
@@ -37,9 +165,190 @@ jobs:
           kubectl describe svc -n istio-system istio-ingressgateway
           kubectl get pods -n knative-serving
           kubectl get pods -n cert-manager
-      - name: Build KServe images
+      - name: Update test overlays
         run: |
-          ./test/scripts/gh-actions/build-images.sh
+          ./test/scripts/gh-actions/update-test-overlays.sh
+          docker image ls
+          cat ./config/overlays/test/configmap/inferenceservice.yaml
+      - name: Install KServe
+        run: |
+          ./test/scripts/gh-actions/setup-kserve.sh
+
+          kubectl get pods -n kserve
+          kubectl describe pods -n kserve
+      - name: Run E2E tests
+        timeout-minutes: 40
+        run: |
+          ./test/scripts/gh-actions/run-e2e-tests.sh "fast or pmml"
```

Comment:
any reason why pmml needs to be separatee?

Reply:
The paddle and pmml images are fairly big approx 2GB each and sometimes, we run out of disk space, if we run all the predictor together...